apps/ui/src/views/Proposal/Votes.vue

```diff
@@ -209,26 +209,39 @@ watch([sortBy, choiceFilter], () => {
               </div>
             </td>
             <td class="relative">
-              <div
-                v-if="proposal.type !== 'basic'"
-                class="truncate"
-                :title="proposal.choices[vote.choice - 1]"
-              >
-                {{ proposal.choices[vote.choice - 1] }}
+              <template v-if="!!props.proposal.privacy && !props.proposal.completed">
+                <div class="hidden md:block">
+                  <div class="flex gap-1 items-center">
+                    <span class="text-skin-heading">Encrypted choice</span>
+                    <i-h-lock-closed class="w-[16px] h-[16px] shrink-0" />
+                  </div>
+                </div>
+                <UiTooltip title="Encrypted choice" class="cursor-help md:hidden">
+                  <i-h-lock-closed class="w-[16px] h-[16px]" />
```

Comment:
```suggestion
                    <IH-lock-closed class="w-[16px] h-[16px] shrink-0" />
                  </div>
                </div>
                <UiTooltip title="Encrypted choice" class="cursor-help md:hidden">
                  <IH-lock-closed class="w-[16px] h-[16px]" />
```

This is convention we follow in this repo.

Reply:
Which is mixing two different conventions btw :D

Reply:
Ideally we should be using `CamelCase` everywhere, even for things like `router-link` https://vuejs.org/style-guide/rules-strongly-recommended.html#self-closing-components

Reply:
is there an eslint rule to enforce this ?

Reply:
I mean `PascalCase`

Reply:
> is there an eslint rule to enforce this ?

Yes, here https://eslint.vuejs.org/rules/component-name-in-template-casing.html